src/us_reverse_geo/Response.js

```diff
@@ -0,0 +1,17 @@
+const Result = require("./Result");
+
+/**
+ * The SmartyResponse contains the response from a call to the US Reverse Geo API.
+ */
+class Response {
+	constructor(responseData) {
+		this.results = [];
+
+		if (responseData !== undefined)
```

Comment:
You can simplify this to be 
```suggestion
		if (responseData)
```